plugins/BEdita/Core/src/Utility/Properties.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2019 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace BEdita\Core\Utility;
+
+use Cake\ORM\TableRegistry;
+use Cake\Utility\Inflector;
+
+/**
+ * Utility class to handle properties in migrations, shell scripts and other scenarios
+ *
+ * Provides static methods to create and remove properties using an array format
+ *
+ * Example:
+ *   [
+ *     [
+ *       'name' => 'custom_one',
+ *       'object' => 'documents',
+ *       'property' => 'boolean',
+ *       'description' => 'my custom description', // optional
```

Comment:
`description` is mentioned here but not used in the method

Reply:
added